learning/web/src/components/CookieBanner/index.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import CookieBanner from "./index"
+import { shallow } from "enzyme"
+
+describe(CookieBanner, () => {
+  it("should render 🍪", () => {
+    const component = shallow(<CookieBanner />)
```

Comment:
Could be worth performing a mount at this point, to catch whether the contents of the cookie banner changes.

Reply:
Not a priority for now - will make separate ticket to come back to this.